aiida/engine/processes/builder.py

```diff
@@ -8,20 +8,25 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 """Convenience classes to help building the input dictionaries for Processes."""
-import collections
+from collections.abc import Mapping, MutableMapping
+import json
 from typing import TYPE_CHECKING, Any, Type
 from uuid import uuid4
 
 from aiida.engine.processes.ports import PortNamespace
 from aiida.orm import Node
+from aiida.orm.nodes.data.base import BaseType
+from aiida.plugins import DataFactory
 
 if TYPE_CHECKING:
     from aiida.engine.processes.process import Process
 
 __all__ = ('ProcessBuilder', 'ProcessBuilderNamespace')
 
+Dict = DataFactory('dict')
```

Comment:
just import this from `aiida.orm`

Reply:
(I would basically advocate for never using these plugin loaders, because they are not statically analysable)

Reply:
Not that I recommend lol (I'm sure I've already talked about this in an aiida meeting); we also "guarantee" what is publicly available to import from `aiida.orm`. Ok maybe if we move e.g. `StructureData` out of core you would have to change the import location, but I think that whole change would be breaking anyway and just a simple find/replace

But anyhow, certainly in core we always import rather than load